### PR TITLE
Juwai's libzmq role not working on mainland connections

### PR DESCRIPTION
**Pull request by zhangjustin**
Modified the zeromq download url ,change the zeromq download source from archive to github.

### CODE REVIEW

**Comment by mark-juwai** [comment]
@zhangjustin This is a public repo, so no ticket information here please.

**Comment by zhangjustin** [comment]
Please review, Thanks

File: defaults/main.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ---
 zeromq_version: 4.0.5
-zeromq_download_url: "https://archive.org/download/zeromq_{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz"
+zeromq_download_url: "https://github.com/zeromq/zeromq4-x/releases/download/v{{ zeromq_version }}/zeromq-{{ zeromq_version }}.tar.gz"
+
```
Comment (mark-juwai) [inline defaults/main.yml]: ? Remove extra blank lines.

**Review by mark-juwai — APPROVED** [review]

**Review by tjoelsson — APPROVED** [review]

**Review by mark-juwai — APPROVED** [review]

**Comment by tjoelsson** [comment]
@zhangjustin can we merge this now?

**Comment by zhangjustin** [comment]
Please merge, thanks

**Pull request merged**